docs/marketplace/tutorials/create-a-rag-application.md

```diff
@@ -150,13 +255,44 @@ Let's try with part of the Mia-Platform documentation:
 }
 ```
 
-We click on the `Execute` button and receive an immediate response indicating that the embedding generation has started. At this point, we simply need to wait for the process to complete. To verify if the process is finished, you can send a request to the `/api/embeddings/status` API. If the response body includes `{ "status": "running" }`, it means that the webpages are still being scraped and embeddings are still being generated. You can call the API multiple times, and when the response includes `{ "status": "idle" }`, then the process is concluded.
+:::info
+You can execute the same request via _curl_ from your terminal with the following command:
+
+```bash
+curl -X POST "https://my-project.console.my-company.com/api/embeddings/generate" \
+-H 'accept: application/json' \
+-H 'Content-Type: application/json' \
+-d "{ \"url\": \"https://docs.mia-platform.eu/docs/fast_data/what_is_fast_data\", \"filterPath\": \"https://docs.mia-platform.eu/docs/fast_data\" }"
+```
+
+:::
+
+We click on the `Execute` button and receive an immediate response indicating that the embedding generation has started.
```

Comment:
above you used "You can execute the .." now you are using "we". I'd strive for more consistency :)

Reply:
You're absolutely right. Done a little bit of rewriting.